### PR TITLE
Redo home page

### PR DESCRIPTION
**Pull request by dasconnor**
I redid our home page to be more tile focused. Definitely open to feedback on links and copy. I'd like to replace the API link with a page on warp messaging when we have a better doc up.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      <main>
+        <br/>
+        <h1 align="center" style={{ fontWeight: '750' }}>Welcome to Avalanche Docs</h1>
+        <section className={styles.features}>
+          <div className="container">
+            <div className="row cards__container">
+              <Card
+                to="subnets/build-first-subnet"
+                header={{
+                  label: "🚀 Launch Your First Subnet",
```
Comment (yulin-dong) [inline src/pages/index.js:22]: Not this PR, however, we would need to consider how to i18n'ize these strings.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: src/pages/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                  label: "🛠️ Launch Your Dapp on Avalanche",
+                }}
+                body={{
+                  label:
+                    "Learn everything you need to deploy an EVM-compatible smart contract",
+                }}
+              />
+
+              <Card
+                to="https://core.app"
```
Comment (yulin-dong) [inline src/pages/index.js:75]: Not sure about this link to core.app, as it does not do an explanation what Core is. https://support.avax.network/en/collections/3391518-core is not a good choice, either.

Comment (dasconnor) [reply src/pages/index.js:75]: I was thinking we wanted a link to each of our most important topics, so I wanted to have one for Core. Not sure what the best page that represents Core is, but that one seemed like the best I could find.

File: src/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                body={{
+                  label:
+                    "Discover how Subnets and Avalanche Consensus are revolutionizing Web3",
+                }}
+              />
+
+              <Card
+                to="nodes"
+                header={{
+                  label: "😎 Become a validator",
```
[the remark this answers is not in the document]
Comment (dasconnor) [reply src/pages/index.js]: fixed this

**Review by yulin-dong — APPROVED** [review]

File: src/pages/index.js (hunk 2 above)
Comment (dasconnor) [reply src/pages/index.js:75]: going to leave as is for now, happy to update later if we have more consensus

**Pull request merged**